apps/studio/components/interfaces/Organization/Usage/Usage.constants.tsx

```diff
@@ -115,14 +115,6 @@ export const USAGE_CATEGORIES: (subscription?: OrgSubscription) => CategoryMeta[
                 name: 'Documentation',
                 url: 'https://supabase.com/docs/guides/platform/database-size',
               },
-              ...(subscription?.usage_based_billing_project_addons === true
```

Comment:
just to double check that we want to remove the disk size section in org usage page?

Reply:
This is just removing a link to disk management for free plan customers. Free plan customers are limited by database size, whereas paying customers are billed on disk size - so yeah change is conscious